app/(golems)/golems/skills/[name]/page.tsx

```
@@ -0,0 +1,363 @@
+      {/* Back link */}
+      <Link
+        href="/golems#skills"
+        className="mb-6 inline-flex items-center gap-1.5 text-sm text-[#8b7355] transition-colors hover:text-[#e59500]"
+      >
+        <svg
+          className="h-4 w-4"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+          strokeWidth={2}
+        >
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)